test/mediumClient_test.js

```diff
@@ -71,20 +71,20 @@ describe('MediumClient - methods', function () {
       authUrl.protocol.should.equal('https:')
       authUrl.hostname.should.equal('medium.com')
       authUrl.pathname.should.equal('/m/oauth/authorize')
-      authUrl.query.should.deepEqual({
+      should.equal(JSON.stringify(authUrl.query), JSON.stringify({
```

Comment:
This is where I changed, others are indentation fixes, deepEqual not working with objects, don't know, are they checking the reference as well. So I had to do JSON.stringify!